piptools/repositories/pypi.py

```diff
@@ -184,13 +184,14 @@ def get_dependencies(self, ireq):
                     force_reinstall=False,
                     ignore_dependencies=False,
                     ignore_requires_python=False,
-                    ignore_installed=True,
+                    ignore_installed=False,
```

Comment:
Why is this changed?

IIRC changing this makes installed packages affect how the requirements are compiled, which is not a good thing.